client/js/screens/Poses/Poses.js

```diff
@@ -1,24 +1,62 @@
 import React from "react";
-import { Text, TouchableOpacity } from "react-native";
+import { View, Text, TouchableOpacity, FlatList, Image } from "react-native";
 import propTypes from "prop-types";
+import styles from "./styles";
 
-const Poses = ({ navigation }) => {
+const thumbnail = {
+  Bear: require("../../assets/images/icons/Bear.png"),
+  Superbug: require("../../assets/images/icons/Superbug.png")
+};
+
+renderSeparator = () => {
+  return (
+    <View
+      style={{
```

Comment:
I have seen this same file in your other pull request. Did you merge your branches?